File: trunk/net.tenteCsharp.templatesProject/src_code/windowManagement/DeviceKind.java
cclass windowManagement.WindowManagement;

public cclass DeviceKind extends TypeEnum {
	
	public DeviceKind (){
		super();		
		values.add("WindowController");		
		values.add("WindowDimmer");		
		values.add("BlindDimmer");		
		values.add("BlindController");
	}
	
}

File: Elio/pseudoCodeGeneratorElio/src-gen/windowManagement/DeviceKind.cs
using System;
using System.Collections;
using System.Text;

namespace SmartHome
{
	/// <summary>
    /// Enumeration
    /// </summary>
	public class DeviceKind : TypeEnum 
	{
		public DeviceKind () 
			: base()
		{		
		values.Add("WindowController");		
		values.Add("WindowDimmer");		
		values.Add("BlindDimmer");		
		values.Add("BlindController");
		}
	
	}
}